docs/Getting Started/Adding the Lock Screen/index.md

```diff
@@ -23,29 +50,70 @@ SDLLockScreenConfiguration *lockScreenConfiguration = [SDLLockScreenConfiguratio
 ```swift
 let lockScreenConfiguration = SDLLockScreenConfiguration.enabled()
 ```
+!@
 
-## Customizing the Provided Lock Screen
-If you would like to use the provided lock screen but would like to add your own appearance to it, we provide that as well. `SDLLockScreenConfiguration` allows you to customize the background color as well as your app's icon. If the app icon is not included, we will use the SDL logo.
+## Customizing the Default Lock Screen
+If you would like to use the provided lock screen but would like to add your own appearance to it, we provide that as well. @![iOS]`SDLLockScreenConfiguration`!@ @![android]`LockScreenConfig`!@ allows you to customize the background color as well as your app's icon. If the app icon is not included, we will use the SDL logo.
 
+@![iOS]
 ![Custom Lock Screen](/assets/CustomLockScreen.png)
+!@
 
+### Custom Background Color
+@![iOS]
 ##### Objective-C
 ```objc
-UIImage *appIcon = <# Retreive App Icon #>
 UIColor *backgroundColor = <# Desired Background Color #>
-SDLLockScreenConfiguration *lockScreenConfiguration = [SDLLockScreenConfiguration enabledConfigurationWithAppIcon:appIcon backgroundColor:backgroundColor];
+SDLLockScreenConfiguration *lockScreenConfiguration = [SDLLockScreenConfiguration enabledConfigurationWithAppIcon:<# Retreive App Icon #> backgroundColor:backgroundColor];
 ```
 
 ##### Swift
 ```swift
-let appIcon: UIImage = <# Retrieve App Icon #>
 let backgroundColor: UIColor = <# Desired Background Color #>
-let lockScreenConfiguration = SDLLockScreenConfiguration.enabledConfiguration(withAppIcon: appIcon, backgroundColor: backgroundColor)
+let lockScreenConfiguration = SDLLockScreenConfiguration.enabledConfiguration(withAppIcon: <# Retrieve App Icon #>, backgroundColor: backgroundColor)
 ```
+!@
 
-## Using Your Own Lock Screen
-If you would like to use your own lock screen instead of the provided SDL one, but still use the logic we provide, you can use a new initializer within `SDLLockScreenConfiguration`:
+@![android]
+```java
+lockScreenConfig.setBackgroundColor(resourceColor); // For example, R.color.black
+```
+!@
 
+### Custom App Icon
+@![iOS]
+##### Objective-C
+```objc
+let appIcon: UIImage = <# Retrieve App Icon #>
+SDLLockScreenConfiguration *lockScreenConfiguration = [SDLLockScreenConfiguration enabledConfigurationWithAppIcon:appIcon backgroundColor:<# Desired Background Color #>];
+```
+
+##### Swift
+```swift
+let appIcon: UIImage = <# Retrieve App Icon #>
+let lockScreenConfiguration = SDLLockScreenConfiguration.enabledConfiguration(withAppIcon: appIcon, backgroundColor: <# Desired Background Color #>)
+```
+!@
+
+@![android]
+```java
+lockScreenConfig.setAppIcon(appIconInt); // For example, R.drawable.lockscreen icon
+```
+!@
+
+@![android]
+### Showing the Device Logo
```

Comment:
```suggestion
### Showing the OEM Logo
```

Reply:
Now that I think about it, the iOS lock screen shows the OEM logo by default, but the dev does not have the ability to disable the logo.

Have an iOS section for **Showing the OEM Logo** that says:
> The default lock screen handles retrieving and setting the OEM logo from head units that support this feature. Currently this feature can not be disabled on the default lock screen.